File: ProductionManagement.Services/Services/Orders/Models/OrderModel.cs
namespace ProductionManagement.Services.Services.Orders.Models
{
    public class OrderModel
    {
        public int Id { get; set; }

        public string OrderName { get; set; }

        public string Description { get; set; }

        public int TankId { get; set; }

        public string TankName { get; set; }

        public int ProductionDays { get; set; }

        public string Color { get; set; }

        public string ProductionLinesNames { get; set; }
    }
}

File: ProductionManagement.WebUI/ClientApp/src/app/modules/orders/models/order-model.ts
export class OrderModel {
  id: number;
  orderName: string;
  description: string;
  tankId: number;
  tankName: string;
  productionDays: number;
  productionLinesNames: string;
  color: string;
  sequence: number;
  startDate?: Date;
  stopDate?: Date;

  checked: boolean;
}
